extmod/modbluetooth.c

```diff
@@ -573,6 +587,8 @@ STATIC mp_obj_t bluetooth_ble_gap_connect(size_t n_args, const mp_obj_t *args) {
         scan_duration_ms = mp_obj_get_int(args[3]);
     }
 
+    bluetooth_handle_uninitialized();
```

Comment:
I don't think this is needed.  It currently raises `OSError(EIO)` if it's not active (on PYBD).